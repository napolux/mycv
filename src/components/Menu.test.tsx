import React from 'react';
import MyCvMenu from './Menu';
import renderer from 'react-test-renderer';
import {
    BrowserRouter as Router,
} from "react-router-dom";

it('renders correctly inside router', () => {
    const tree = renderer
        .create(
            <Router>
                <MyCvMenu />
            </Router>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});