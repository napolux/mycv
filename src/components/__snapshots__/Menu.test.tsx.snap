// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly inside router 1`] = `
<div
  className="ui container"
>
  <div
    className="ui menu"
  >
    <div
      className="header item"
      onClick={[Function]}
    >
      MyCv
    </div>
    <div
      className="item"
      onClick={[Function]}
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </div>
    <div
      className="item"
      onClick={[Function]}
    >
      <a
        href="/login"
        onClick={[Function]}
      >
        Login
      </a>
    </div>
  </div>
</div>
`;
